src/backend/distributed/sql/udfs/citus_stat_activity/11.0-1.sql

```diff
@@ -0,0 +1,13 @@
+DROP VIEW IF EXISTS pg_catalog.citus_stat_activity;
+
+CREATE VIEW citus.citus_stat_activity AS
+SELECT global_pid, nodeid, pg_stat_activity_from_all_nodes.* FROM
+run_command_on_workers($$
+    SELECT to_json(array_agg(pg_stat_activity.*)) FROM pg_stat_activity
+$$) as text_result_from_all_nodes
```

Comment:
this should include the coalesce discussed here: https://github.com/citusdata/citus/issues/3172#issuecomment-1047562173

Otherwise the query fails if any node returns zero rows. With pg_stat_activity that cannot happen, but still